DREAM3DReviewFilters/SliceTriangleGeometry.cpp

```diff
@@ -359,11 +366,12 @@ void SliceTriangleGeometry::determineBoundsAndNumSlices(float& minDim, float& ma
       maxDim = m_Zend;
     }
   }
-
-  m_NumberOfSlices = static_cast<MeshIndexType>((maxDim - minDim) / m_SliceResolution) + 1;
+  // TODO: Is this still correct? Why have the subtraction at all?
+  m_NumberOfSlices = static_cast<MeshIndexType>((maxDim - 0.0) / m_SliceResolution) + 1;
```

Comment:
I think this is right. but I'm a little confused looking over the code.  Do we have an example of an STL file where it was crashing?  Would be good for a unit test.

Reply:
I think we could certainly create a small STL file that is lifted off the Z=0 plane as a test.